src/main/java/com/minecolonies/coremod/inventory/IInteractiveItemHandler.java

```diff
@@ -0,0 +1,9 @@
+package com.minecolonies.coremod.inventory;
+
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraftforge.items.IItemHandler;
+
+public interface IInteractiveItemHandler extends IItemHandler {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Move this left curly brace to the beginning of next line of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Document this public interface. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUndocumentedApi)